### DISCUSSION TITLE
Get field of json - can't figure out json-path

### DISCUSSION THREAD

**Discussion by webwurm** [AntlerVC/firetable#964] [Support / Q&A] [answered]
I have a problem that I just can't figure out - so I hope the crowd can help me.

I have a field, called "tags", with json in it, like so:
<img width="293" alt="json01" src="https://user-images.githubusercontent.com/1053176/203022744-f2116388-1ad8-46d4-a9b6-504ee64c59ad.png">

In a derivative field I want to this field:
`return row.tags.$[0]._key.path.segments[6];`

...but that doesn't work. Nor does:
`return row.tags.[0]._key.path.segments[6];`
`return row.tags.root[0]._key.path.segments[6];`
`return row.tags.x[0]._key.path.segments[6];`

How do I access the path, when the first element is already an array?

Thanks a lot!

**Comment by shamsmosowi** [comment, marked as answer]
try `return row.tags[0]._key.path.segments[6]`

**Reply by webwurm** [reply]
Sadly that doesn't work. It throws the error `Cannot read properties of undefined (reading 'path')`

**Reply by shamsmosowi** [reply]
is this a firestore document reference?

**Reply by webwurm** [reply]
It's an array of references (see [here](https://github.com/rowyio/rowy/discussions/973) in more detail)

**Reply by BeeBombshell** [reply]
Hey guys, closing this discussion. Follow the discussion https://github.com/rowyio/rowy/discussions/973 for updates on this.
